app/frontend/src/components/forms/PrintOptions.vue

```diff
@@ -345,13 +355,19 @@ export default {
           );
         }
         this.isValidFile = true;
+        this.isValidSize = true;
       }
     },
 
     handleFileUpload(event) {
       this.fileInputKey += 1;
       this.templateForm.files = event;
-      this.validateFileExtension(event);
+      this.validateFile(event);
+
+      // need to force immediate validation as the v-file-input is bound to the :key
+      this.$nextTick(() => {
+        this.$refs.fileInput.validate();
+      });
```

Comment:
In my previous changes to this file, I had added a new `:key` attribute to the v-file-input, which resolves another [issue](https://bcdevex.atlassian.net/browse/FORMS-1049).  This created a bug where the file validation was not triggered immediately. 

I have added this code to apply the validation rules to the component immediately, fixing the issue.